src/jobs/rss-news.js

```diff
@@ -61,17 +62,30 @@ async function processSchool(school) {
 }
 
 async function run() {
+	logger.info('will update rss feeds...');
+	const errors = [];
 	await database.connect();
 
 	const cursor = schoolModel.find({}).cursor();
 	let school = await cursor.next();
 	while (school) {
-		await processSchool(school);
+		try {
+			await processSchool(school, errors);
```

Comment:
To use await on this position and not a stack of promises with await Promise.all is for slow down the process by many rss feeds? But maybe we can use chunks from 20 - 50 rss feeds, at the same time?
Is not your code, and in the scope from this ticket. But it looks like it's a performance lag and therefore a code mistake from Lukas.